canvas/image.go

```diff
@@ -107,6 +109,17 @@ func NewImageFromURI(uri fyne.URI) *Image {
 		}
 	}
 
+	if uri.Scheme() == "http" || uri.Scheme() == "https" {
```

Comment:
Do we need to do this?
I know that HEAD is there to help us avoid downloading data, but in this case the developer probably knows it exists already. So how about we do the Go thing and just make the GET request and handle error if isn't there?

Reply:
That makes sense. I was thinking more from a HTTP perspective.